core/src/main/java/com/epam/esm/util/Status.java

```diff
@@ -0,0 +1,26 @@
+package com.epam.esm.util;
+
+public enum Status {
+    SUCCESSFUL(200, "Successful"),
+    GIFT_NOT_FOUND(1001, "Gift not found"),
```

Comment:
please read a suggestion for custom error codes from task's description and update your codes;)

Reply:
do you need field message in this class? it not, remove it